src/tests/__tests__/Checkbox.react-test.js

```diff
@@ -15,4 +15,14 @@ describe('Checkbox Component', () => {
     tree.props.checked = true;
     expect(tree).toMatchSnapshot();
   });
+
+  test('no fancy style', () => {
+    const component = shallow(<Checkbox>Test Checkbox</Checkbox>);
```

Comment:
btw, can't we use snapshots here as well?

Reply:
Yes, we can. Updated!